packages/devextreme/scss/widgets/fluent/toolbar/_index.scss

```diff
@@ -33,6 +33,10 @@
       height: $fluent-toolbar-height;
     }
   }
+
+  .dx-toolbar-item .dx-item-content {
+    display: flex;
```

Comment:
I believe we can't afford such a general solution. There's no justification for making all content containers flex-based. Besides, the content aligns correctly at the center. I suggest looking for a solution at the switch level in the toolbar item and/or fix issues in the switch component styles.